Day7-Assignment/src/Components/AboutUs.js

```diff
@@ -0,0 +1,9 @@
+const AboutUs =()=>{
```

Comment:
Can name your "Components" folder as "components". Keeps the Component folders distinguishable from normal folders.

Reply:
> Nicely written. Kudos Rasika!!